test/unit/master/test_cluster_runner_config.py

```diff
@@ -55,6 +55,15 @@ class TestClusterRunnerConfig(BaseUnitTestCase):
     PHPUnit:
     """
 
+    _VALID_CONFIG_WITH_EMPTY_COMMAND = """
+    PHPUnit:
+        commands:
+            - echo "go"
+            -
```

Comment:
Should we add a comment to the empty line (or add an additional line with a comment?)

An example of what I mean is:
```
commands:
    - echo "go"
    - # some dumb comment
    - 
```

The reason I suggest this is related to the specific use case listed in the issue ticket #122. It looks like that syntax for a comment gets interpreted as a yaml comment and ignored instead of being interpreted as a bash comment and included in the final command. (This is a good thing since I think a bash comment would break something! :smile:) But I feel like we'd want to make sure we keep that behavior if we ever change yaml libraries or something like that.